packages/terra-dropdown-button/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ ChangeLog
 
 Unreleased
 ----------
+### Fixed
+* Fixes Primary click button not closing menu, when menu is open
+* Fixes multiple instances of split-button and drop-down opening simultaneously.
```

Comment:
nit: These read like a PR description. We don't have a consistent standard for entries, but generally they either start with `Fixed` or exclude it under the section title.

```
### Fixed
* Fixed primary button click not closing menu when menu is open
```

```
### Fixed
* Primary button click not closing menu when menu is open
```

Reply:
Updated here b0270084ef21c8948464dc3e855587866964f1f9